addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSvnScanRule.java

```diff
@@ -693,14 +647,14 @@ private boolean findSourceCodeSVN(HttpMessage originalMessage) throws Exception
                             }
                         } else
                             throw new SQLException(
-                                    "Could not open a JDBC connection to SQLite file "
-                                            + tempSqliteFile.getAbsolutePath());
+                                    "Could not open a JDBC connection to SQLite file {}",
```

Comment:
Not on purpose? This is an exception.

Reply:
Whoops went overboard looking at plus concatenations.